src/tests/ngraph_helpers/snippets_ngraph_functions/src/subgraph_matmul.cpp

```diff
@@ -29,6 +29,17 @@ std::shared_ptr<ov::Model> MatMulSinhFunction::initReference() const {
                                                                       ParameterVector{indata0, indata1}));
     return std::make_shared<ov::Model>(NodeVector{matmul}, ParameterVector{data0, data1});
 }
+std::shared_ptr<ov::Model> MatMulBiasSinhFunction::initOriginal() const {
```

Comment:
The same suggestion as for the Select-Broadcast PR: it might be easier to create `Sinh`-less functions and use internal config to bypass `SnippetsMarkSkipped`. This will make the rebase easier.

Here, below and also in subgraph_mha.cpp